isos/bootstrap.sh

```diff
@@ -88,34 +87,66 @@ PKGDIR=$(mktemp -d)
 
 unpack $package $PKGDIR
 
+# load the repo to use from the package if not explicit in env
+REPO=${REPO:-$(cat $PKGDIR/repo.cfg)}
+REPODIR="$DIR/base/repos/${REPO}/"
+PACKAGE_MANAGER=${PACKAGE_MANAGER:-$(cat $REPODIR/repo-spec.json | jq -r '.packagemanager')}
+PACKAGE_MANAGER=${PACKAGE_MANAGER:-tdnf}
+setup_pm $REPODIR $PKGDIR $PACKAGE_MANAGER $REPO
+
+
 #selecting the init script as our entry point.
 if [ -v debug ]; then
```

Comment:
Think we should drop the bootstrap-debug for now - it's not been maintained and I don't know that it's still functional.